src/main.rs

```diff
@@ -1,7 +1,8 @@
 use axum::Router;
 use tokio::net::TcpListener;
 use tower_http::cors::{Any, CorsLayer};
-use tracing::error;
+use tracing::{error, info};
```

Comment:
```suggestion
use tracing::{error};
```

`tracing::info` is unused.